config/default/manager/manager.yaml

```diff
@@ -31,7 +31,7 @@ metadata:
 spec:
   strategy:
     type: Recreate
-  replicas: 2
+  replicas: 1
```

Comment:
Maybe now we can go back to `RollingUpdate` ?

Reply:
That may be dangerous. We don't have locking and we don't want the old pod to run at the same time as the new one.